nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/AbstractMongoProcessor.java

```diff
@@ -373,6 +374,11 @@ protected Collection<ValidationResult> customValidate(ValidationContext context)
         boolean clientIsSet = context.getProperty(CLIENT_SERVICE).isSet();
         boolean uriIsSet    = context.getProperty(URI).isSet();
 
+        if (uriIsSet) {
+            DeprecationLoggerFactory.getLogger(AbstractMongoProcessor.class).warn("The MongoDB URI and other client " +
+                    "configuration properties on this proessor are deprecated and will be removed in NiFi 2.0");
```

Comment:
The message should indicate that the MongoClientService should be used. The `will be removed in NiFi 2.0` portion of the message should be removed since it is implicit in all deprecation warnings.
```suggestion
                    "connection properties should be replaced with an implementation of the MongoClientService");
```